app/src/main/java/com/duckduckgo/mobile/android/duckduckgo/data/base/JsonEntity.java

```diff
@@ -0,0 +1,17 @@
+package com.duckduckgo.mobile.android.duckduckgo.data.base;
+
+/**
+ * Created by fgei on 6/12/17.
+ */
+
+public abstract class JsonEntity {
```

Comment:
Looks great, I'd call the method `create` to avoid any confusion with the class level `newInstance` method